shipping/uc_quote/address_quote.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * UcAddressQuote class
+ */
+
+/**
+ * Defines an object to hold Ubercart address for shipping quotes on products.
+ */
+class UcAddressQuote extends UcAddress {
+  /**
+   * @var int
+   */
+  public $nid;
+
+  /**
+   * @inheritdoc
+   */
+  public function __construct($country = NULL) {
+    parent::__construct($country);
```

Comment:
Since this just calls the parent, we don't need to override it.

Reply:
Yeah. The class property just looked lonely so I had added it. ;)

Reply:
I've removed it